source/MaterialXView/Editor.cpp

```diff
@@ -21,6 +24,55 @@ class EditorFormHelper : public ng::FormHelper
     void setVariableSpacing(int val) { mVariableSpacing = val; }
 };
 
+// Custom color picker so we can get numeric entry and feedback.
+//
+class MyColorPicker : public ng::ColorPicker
```

Comment:
Took me a few iterations but it ended up being pretty simple to add in the float entries.